spec/frameSpec.js

```diff
@@ -0,0 +1,47 @@
+describe('Frame', function() {
+
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('when playing', function() {
```

Comment:
This is a superfluous `describe`; since there is no other context, 'when playing' is implicit.  Furthermore, 'when playing' is a context rather than a feature you are describing, so should be in a `context` method instead.  It's a semantic quibble, but nevertheless important.